Sources/ArcGISToolkit/Components/Augmented Reality/CalibrationView.swift

```diff
@@ -19,140 +19,150 @@ import ArcGIS
-                                rotateHeading(by: joystickHeadingDelta)
-                            }
-                            headingTimer = timer
-                            // Add the timer to the main run loop.
-                            RunLoop.main.add(timer, forMode: .default)
-                        }
-                        
-                        Button {
-                            rotateHeading(by: 1)
-                        } label: {
-                            Image(systemName: "plus")
-                                .imageScale(.large)
-                        }
-                    }
+                HStack(alignment: .firstTextBaseline) {
+                    Text("Calibration")
+                        .font(.title)
+                        .lineLimit(1)
+                    Spacer()
+                    dismissButton
+                        .layoutPriority(1)
                 }
-                VStack {
-                    Text("Elevation: \(viewModel.calibrationElevation?.rounded(.towardZero) ?? viewModel.cameraController.originCamera.location.z?.rounded(.towardZero) ?? 0, format: .number) m")
-                    HStack {
-                        Button {
-                            updateElevation(by: -1)
-                        } label: {
-                            Image(systemName: "minus")
-                                .imageScale(.large)
-                        }
-                        Slider(value: $elevationSliderValue, in: -20...20) { editingChanged in
-                            if !editingChanged {
-                                elevationTimer?.invalidate()
-                                elevationTimer = nil
-                                elevationSliderValue = 0.0
-                            }
-                        }
-                        .onChange(of: elevationSliderValue) { elevation in
-                            guard elevationTimer == nil else { return }
-                            // Create a timer which rotates the camera when fired.
-                            let timer = Timer(timeInterval: 0.1, repeats: true) { _ in
-                                updateElevation(by: joystickElevationDelta)
-                            }
-                            elevationTimer = timer
-                            // Add the timer to the main run loop.
-                            RunLoop.main.add(timer, forMode: .default)
-                        }
-                        Button {
-                            updateElevation(by: 1)
-                        } label: {
-                            Image(systemName: "plus")
-                                .imageScale(.large)
+                .padding(.bottom)
+                headingSlider
+                Divider()
+                elevationSlider
+            }
+            .padding()
+            .background(.regularMaterial)
+            .clipShape(RoundedRectangle(cornerRadius: 15))
+            .frame(maxWidth: 350)
```

Comment:
Shall we pick a slightly bigger frame size, so that the view always take the full width minus the padding on an iPhone portrait orientation, and be a fixed size on iPad? From the [resolution list](https://developer.apple.com/design/human-interface-guidelines/layout#Specifications) it seems 414 (iPhone 15 PM's 430 - 2 * 8) can be a good value.

In this way, the leading padding for the button and calibration view will be consistent when the button is placed on leading or trailing edge.